content/en/real_user_monitoring/flutter/mobile_vitals.md

```diff
@@ -39,18 +39,25 @@ The following metrics provide insight into your mobile application's performance
+| Memory utilization             | High memory usage can lead to [out-of-memory crashes][10], which causes a poor user experience. <br /><br />  RUM tracks the amount of physical memory used by your application in bytes for each view, over the course of a session. The recommended range is <200MB for good and <400MB for moderate. <br /><br />  You can see the top views with the most memory consumption on average over a selected time period under **Mobile Vitals** in your application's Overview page.                                                                                                                                                                                                                                                                                            |
+| Widget build time             | This is the duration of time taken build the frame on the UI thread. To ensure a smooth animations, this should not exceed 16ms for 60 FPS, and 8ms for 120 FPS. <br /><br />  High values here mean you need to look into optimizing your build methods for this view. See [Control Build Cost][11] in the Flutter documentation.                                                                                                                                                                                                                                                                                            |
+| Raster time             | This is the duration of time taken to rasterize the frame on the raster thread. To ensure a smooth animations, this should not exceed 16ms for 60 FPS, and 8ms for 120 FPS. <br /><br />  High values here may mean your view is complex to render. See [Identifying Problems in the GPU Graph][12] in the Flutter documentation.                                                                                                                                                                                                                                                                                            |
 ## Further reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /real_user_monitoring/ios/mobile_vitals/
 [2]: /real_user_monitoring/android/mobile_vitals/
-[3]: https://docs.flutter.dev/perf/best-practices#control-build-cost
-[4]: https://docs.flutter.dev/perf/ui-performance#identifying-problems-in-the-gpu-graph
+[3]: https://docs.flutter.dev/perf/ui-performance
+[4]: https://docs.flutter.dev/tools/devtools/performance
+[5]: https://developer.android.com/topic/performance/vitals/frozen [ NEED TO REPLACE WITH FLUTTER DOC LINK IF STILL RELEVANT ]
```

Comment:
I think it's fine to use the Android link here. It's the only documentation that I know of that explains this (iOS uses the Android documentation for this as well).